app/models/authorization.rb

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
+
+class Authorization
+  include HalApi::RepresentedModel
+
+  attr_accessor :token
+
+  def initialize(token)
+    @token = token
+  end
+
+  def user_id
+    token.user_id
+  end
+
+  def to_model
+    self
+  end
+
+  def persisted?
+    false
+  end
+
+  def cache_key
```

Comment:
I don't think it is or should be, but not knowing exactly how this model may be used in the future, I wanted to provide a sensible implementation for it, since it is a standard method for a hal represented model